Makefile

```diff
@@ -46,6 +47,8 @@ dist:
 	make clean
 	python plugins/apps/threatbus_misp/setup.py sdist bdist_wheel
 	make clean
+	python plugins/apps/threatbus_vast/setup.py sdist bdist_wheel
+	make clean
```

Comment:
This whole dependency on `make clean` could be expressed a lot cleaner with more idiomatic make, but I suppose it's fine for what it is.